eng/Versions.props

```diff
@@ -62,6 +62,7 @@
     <SystemSecurityCryptographyXmlVersion>9.0.0-rc.2.24473.5</SystemSecurityCryptographyXmlVersion>
     <SystemTextEncodingsWebVersion>9.0.0-rc.2.24473.5</SystemTextEncodingsWebVersion>
     <SystemTextJsonVersion>9.0.0-rc.2.24473.5</SystemTextJsonVersion>
+    <SystemTextJson8Version>8.0.5</SystemTextJson8Version>
```

Comment:
If STJ revs, e.g. with a security patch, we'll need to remember to update this, or will it be automatically updated by darc? Do we need a corresponding entry in Version.Details.xml?

Reply:
It's not clear to me that it would be updated automatically. Maybe @ViktorHofer knows.

Reply:
Version.Details.xml is the input source for Maestro dependency flow which handles repo -> repo internal flow. If you want to get a version updated based on a new package being available on nuget.org, you would need to use Dependabot.

Reply:
Forgive my ignorance but if this extension doesn't have a dependency on .Net 9, why the package is a `9.x-preview` and not a `8.x-preview` (targeting NET 8 dependencies only) ?

Reply:
Which package are you asking about?